### PR TITLE
[JS] Bug fix for getNationalSignificantNumber

### PR DESCRIPTION
**Pull request by lararennie**
Fixing bug where getNationalSignificantNumber for an empty phone number object would return the string null in Javascript.

---
This change is [Reviewable](https://reviewable.io/reviews/googlei18n/libphonenumber/1821)

### CODE REVIEW

File: javascript/i18n/phonenumbers/phonenumberutil.js [hunk 1]
```diff
@@ -2430,10 +2430,13 @@ i18n.phonenumbers.PhoneNumberUtil.prototype.
...
     function(number) {
 
-  // If leading zero(s) have been set, we prefix this now. Note this is not a
-  // national prefix.
+  if (!number.hasNationalNumber()) {
+    return '';
+  }
   /** @type {string} */
   var nationalNumber = '' + number.getNationalNumber();
+  // If leading zero(s) have been set, we prefix this now. Note these are not a
```
Comment (keghani) [inline javascript/i18n/phonenumbers/phonenumberutil.js]: these are --> this is?
alternatively, change the whole thing:

`we prefix these now. Note these do not make up a national prefix.`

this -> these
there are not a -> these do not make up a

Comment (lararennie) [inline javascript/i18n/phonenumbers/phonenumberutil.js]: Changed the whole thing :)

**Comment by lararennie** [comment]
Added pending_code_changes.txt too.

**Comment by keghani** [comment]
Still LGTM. Thanks for remembering about the demo!

**Pull request merged**